applications/hybrid-translation.md

```diff
@@ -0,0 +1,15 @@
+---
+description: Hybrid machine translation
+---
+
+# Hybrid translation
+
+Hybrid translation is a balance between raw machine translation and post-edited text.
```

Comment:
Contrasts like this should correspond.

> Hybrid translation is a balance between pure machine translation and human post-edited translation.

This is true, but it sounds like it also includes PE MT.  So I think it's good to be clear on how it's different than PE-MT.  Something like:

> With hybrid translation, some machine translations are never seen or touched by a human translator.